### ISSUE TITLE
VError: Failed after 5 retries after initial install of Kalabox and connection to Panethon

### ISSUE THREAD

**Issue by scottfatbeehive** [kalabox/kalabox#1761]
Bug Report
==========

**Tell us about your setup**

What is your Kalabox version and operating system?
Kalabox 2.1.3 on Ubuntu 16.04

Have you previously installed Kalabox? If so, what version?
No

**Tell us about your bug**

Give a brief description of what happened, including any error messages that appeared.

I installed Kalabox via the .deb file. I added a new wordpress site not connected to Pantheon without issue. It spun up and worked correctly. When I tried to connect a Pantheon one, Kalabox stopped at 'Starting Containers' with the nofitication 'VError: Failed after 5 retries. {"max":5,"backoff":500}:' 

Please detail the steps we can take to replicate this bug. Be as detailed as possible.

- I created a new fresh wordpress site via Pantheon and made no changes to it
- I then added my pantheon machine key and hit add site in Kalabox
- A few steps automatically completed, but Kalabox stopped at 'Starting Containers' with the nofitication 'VError: Failed after 5 retries. {"max":5,"backoff":500}:'
- Looking in the terminal, here is the last occurrence of one of the retries:

```
debug: SPAWN DOCKER-COMPOSE ==> Spawn exited with code: 1
verbose: DOCKER COMPOSE ==> Running: ["--project-name","fatbeehivecom-test2","--file","/home/scott/Kalabox/fatbeehivecom-test2/kalabox-compose.yml","--file","/home/scott/Kalabox/fatbeehivecom-test2/kalabox-cli.yml","--file","/home/scott/.kalabox/downloads/fatbeehivecom-test2/fatbeehivecom-test2-2.yml","--file","/home/scott/.kalabox/downloads/fatbeehivecom-test2/fatbeehivecom-test2-3.yml","up","-d"]
```
- Here is that kalabox-compose.yml file it is attempting to run (added .txt to allow upload to github issue)
[kalabox-compose.yml.txt](https://github.com/kalabox/kalabox/files/687200/kalabox-compose.yml.txt)

Please detail the steps we can take to replicate this bug. Be as detailed as possible.

Does this bug prevent you from using Kalabox?
Yes

**Comment by pirog** [comment]
@scottfatbeehive a little bit of a correction here:

you can run docker side by side with kalabox (we use a custom docker daemon) but you cant have another service bound to port 80 eg apache (this could be with docker or could just be native).

**Comment by scottfatbeehive** [comment]
Makes sense, I must have something tied to my local docker setup; I am routing my docker containers through apache so perhaps with docker service on in my case, apache2 service is also firing up. Thanks for the info!
